apiserver/facades/client/application/application.go

```diff
@@ -2751,11 +2751,22 @@ func (api *APIBase) ApplicationsInfo(in params.Entities) (params.ApplicationInfo
 			continue
 		}
 
+		var channel string
+		origin := app.CharmOrigin()
+		if origin != nil && origin.Channel != nil {
+			if origin.Channel.Track != "" {
+				channel = fmt.Sprintf("%s/", origin.Channel.Track)
+			}
+			channel = fmt.Sprintf("%s%s", channel, origin.Channel.Risk)
```

Comment:
Can we use `corecharm.Channel{Track: origin.Channel.Track, Risk: origin.Channel.Risk}`, then we can use `Normalize()` and then `String()`. That way we don't duplicate the code here and there.